hack/ci/entrypoint.sh

```diff
@@ -92,8 +92,12 @@ function run_tests() {
     mkdir -p "$ARTIFACT_DIR"
     # required to be set for test binary
     export ARTIFACTS=${ARTIFACT_DIR}
+    # addresses for testing external connection
```

Comment:
I'm unable to find who is using this. Do you know?

Reply:
openshiftCI is using this.
We agreed offline that this can be removed, so I removed it.